src/pages/Landing/Landing.jsx

```diff
@@ -0,0 +1,47 @@
+import { Link } from "react-router-dom";
+import "./Landing.css";
+import hero from "../../data/image/hero.svg";
+import { useDocumentTitle } from "../../customHooks";
+import { ThemeToggle } from "../../components";
+import { useAuth } from "../../contexts";
+export const Landing = () => {
+	useDocumentTitle("Landing | MS");
+
+	const { isAuth } = useAuth();
+
+	const getBtnText = (() => (isAuth ? "Create New Note" : "Join Now"))();
+
+	const getLinkPath = (() => (isAuth ? "/home" : "/signup"))();
+
+	return (
+		<div className="landing_page main">
+			<section className="content_section">
+				<div className="app_name h1">
+					<span className="color--primary">Magnificent</span> Notes
+				</div>
+				<div className="app_tagline h4">
+					Meet your modern
+					<br />
```

Comment:
don't use br tag its not semantic and not used anymore

Reply:
Ok. will update.